src/Aquifer.API/Endpoints/Help/Endpoint.cs

```diff
@@ -0,0 +1,43 @@
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Help;
+
+public class Endpoint(AquiferDbContext dbContext) : EndpointWithoutRequest<Response>
+{
+    public override void Configure()
+    {
+        Get("/help");
```

Comment:
I think this might be a little too generic, and there could be other forms of "help" in the future. We might have such a thing for Well in the future, for example. So I would probably go a bit further and say `/help/aquifer-cms/documents`, and have the file folder structure / namespace match that as well.